cmake/sdl.cmake

```diff
@@ -0,0 +1,59 @@
+#===============================================================================
+# Copyright 2017-2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+# Manage secure Development Lifecycle-related compiler flags
+#===============================================================================
+
+if(SDL_cmake_included)
+    return()
+endif()
+set(SDL_cmake_included true)
+
+if(UNIX)
+    set(CMAKE_CCXX_FLAGS "-fPIC -Wformat -Wformat-security")
```

Comment:
What's the difference between `CMAKE_CCXX_FLAGS` and `CMAKE_CXX_FLAGS`?

Reply:
I stole this from mkldnn. It is just an internal, poorly named variable. I will fix it and other "custom" variables. Took me a minute to see what they were doing too.